kategory-core/src/test/kotlin/kategory/data/OptionTTest.kt

```diff
@@ -2,17 +2,26 @@ package kategory
 
 import io.kotlintest.KTestJUnitRunner
 import io.kotlintest.properties.forAll
-import kategory.laws.FunctorFilterLaws
-import kategory.laws.TraverseFilterLaws
 import io.kotlintest.matchers.shouldNotBe
 import org.junit.runner.RunWith
 
 @RunWith(KTestJUnitRunner::class)
 class OptionTTest : UnitSpec() {
+
     val EQ_ID: Eq<HK<OptionTKindPartial<IdHK>, Int>> = Eq { a, b ->
         a.value() == b.value()
     }
 
+    val EQ_OPTION = object : Eq<HK<OptionTKindPartial<OptionHK>, Int>> {
```

Comment:
There's a new, easier constructor. See `Eq { a, b -> a.value() == b.value() }` and let the type inference do the rest!